packages/ui-portal/src/Portal/index.tsx

```diff
@@ -34,15 +34,39 @@ import { bidirectional, BidirectionalProps } from '@instructure/ui-i18n'
 /**
  * The DOM Node of the Portal. It is created as a `span` element.
  */
-type PortalNode = HTMLSpanElement | null
+type PortalNode = HTMLSpanElement | null | undefined
 
 type OwnProps = {
+  /**
```

Comment:
Moved comments here from PropTypes, we should keep them here if we want to get rid of PropTypes eventually (this could be a step in the ts typing.)

Reply:
yep, I agree